dags/destinations/ga4mp.py

```diff
@@ -351,8 +351,8 @@ def schema() -> Optional[ProtocolSchema]:
         [
             ("api_secret", str, Field(description="An API SECRET generated in the Google Analytics UI.")),
             ("event_type", Literal["gtag"] | Literal["firebase"], Field(description="GA4 client type.")),
-            ("measurement_id", str, Field(condition="event_type==gtag", description="The measurement ID associated with a stream. Found in the Google Analytics UI.")),
-            ("firebase_app_id", str, Field(condition="event_type==firebase", description="The Firebase App ID. The identifier for a Firebase app. Found in the Firebase console.")),
+            ("measurement_id", str, Field(condition_field="event_type", condition_target="gtag", description="The measurement ID associated with a stream. Found in the Google Analytics UI.")),
```

Comment:
do you think this is the only type of condition we'll be dealing with, e.g. equality? or is it worth introducing something like `condition_type="EQUALITY"`?
maybe overkill...

Reply:
I can't think about other conditions right now, so yeah, if we need to extend in the future something like `condition_type` would be easy to add.